### PR TITLE
[PVR][Estuary] PVR channel manger dialog GUI adjustments

### PR DESCRIPTION
**Pull request by djp952**
## Description
This is a minor/cosmetic-only PR that primarily impacts the Estuary skin, but does have a very small code change included to support it.  The basic gist is that I found the PVR Channel Manager dialog box to have a few layout/visual concerns that I felt would be annoying from an end user's perspective:

- The dialog was not tall enough to accommodate all three of the possible "Misc options" buttons without some clipping.
- The channel list control was not aligned to the height of the list entries, which could cause a partial list item to be displayed.
- The "slide" animation used when the "New channel" button wouldn't be available was kind of weird looking/awkward to me; I felt this would look better as a disabled button that is always present in the dialog rather than hiding it from the user and causing animation weirdness if they toggle between Radio/TV addons that don't all support the same feature set.

I am happy to remove/add anything deemed necessary here or is deemed a "personal preference" that most users will not get any benefit from.

## Motivation and context
Please see above; this PR is a few GUI suggestions that would not impact functionality if it's either approved or discarded.

NOTE: I have a number of proposed changes to this dialog that I'm working on as time allows'; this PR is very "low-hanging fruit" to try and baseline the cosmetic aspect of the work.

## How has this been tested?
Tested on Windows 10 x64 Desktop (20H2), with a display set to 120 DPI.  A Windows system with a 4K display is available if necessary to verify the proposed changes., but Kodi scales controls/dialogs very well and do not anticipate any visual difference.

## What is the effect on users?
The intended effect for end users is that dialog controls will be better aligned and the seemingly strange animation effect used to add the "New channel" button will be replaced with a static (yet disabled) control.

## Screenshots (if appropriate):

BEFORE CHANGE:

This view illustrates the primary problem the PR intends to address, note that when all three buttons are present the bottom button is clipped and the group itself gets a little bunched up.  In the as-is the NEW CHANNEL button is hidden by default and when the user switches to either Radio or TV and that button would be visible, the animation is a bit weird looking.  Also (very minor thing) highlights that the channel list view is a tad too big.  Since the PR makes the dialog/list view bigger it was more pronounced, which is why it was mentioned (see image below):

before

AFTER CHANGE:

This is a view of the proposed layout that illustrates the disabled NEW CHANNEL button as opposed to it not being present as well as the alignment of the list view and the lower button.  **NOTE:** there is an unrelated state issue with the dialog box that causes stale information to show as "Channel options", this has nothing to do with the skin and is intended to be addressed separately:

after-tv

This is a view of the proposed layout with a fully populated list view and with the NEW CHANNEL button enabled. It also illustrates that the list view is sized such that no partial entry will be present:

after-radio

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [X] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by jjd-uk** [comment]
When creating skin PR's it would be helpful if you included before and after screenshots.

**Comment by djp952** [comment]
Agreed, I ran out of time last night and wasn't able to get back to this until just now, family thing came up.  Please let me know if any additional screen shots are desired!

**Review by ksooo — COMMENTED** [review]
I'm not a skinner, but from what I can see this looks correct.

**Comment by ksooo** [comment]
@ronie mind taking a look at the skin changes?

**Comment by djp952** [comment]
PR was updated to include a few changes to <onleft>/<onright> to improve keyboard navigation and change an incorrect <description> tag that would not have been relevant to a subsequent PR that is to be issued, and I didn't want it rejected for this; these additional changes are non-critical "adjustments" that fit better in this PR in my opinion.

**Comment by ksooo** [comment]
@ronie ping

File: addons/skin.estuary/xml/DialogPVRChannelManager.xml [hunk 1]
```diff
@@ -30,18 +30,18 @@
...
 					<left>20</left>
 					<top>30</top>
 					<width>630</width>
-					<height>710</height>
+					<height>770</height>
 					<onup>20</onup>
 					<ondown>20</ondown>
 					<onleft>9000</onleft>
-					<onright>60</onright>
+					<onright>9002</onright>
```
Comment (ronie) [inline addons/skin.estuary/xml/DialogPVRChannelManager.xml:44]: it's now not possible to focus the scrollbar anymore, so please revert this change

File: addons/skin.estuary/xml/DialogPVRChannelManager.xml [hunk 2]
```diff
@@ -139,11 +139,11 @@
 				<left>700</left>
 				<control type="grouplist" id="9002">
 					<top>80</top>
-					<onleft>60</onleft>
+					<onleft>20</onleft>
```
Comment (ronie) [inline addons/skin.estuary/xml/DialogPVRChannelManager.xml:142]: same here, please revert

**Review by ronie — APPROVED** [review]
thx, looks good to me overall, apart from the navigational changes.

**Comment by djp952** [comment]
Requested navigational reverts made (sorry for the delay); I see this has been tagged for backport if accepted, would those navigational changes still be applicable for Matrix or just don't do that?

**Comment by ksooo** [comment]
Only bug fixes for Matrix, please. No improvements, cleanups or features.

**Comment by djp952** [comment]
Are there any additional changes required to this? Just want to make sure that everything to date is in order before (finally) getting to the PR(s) I really want to propose for the PVR channel manager system; I would prefer to base those on a clean and fully merged baseline if possible.  Thank you!

**Review by ksooo — APPROVED** [review]
